pkg/util/util.go

```diff
@@ -266,3 +268,74 @@ func CopyDir(source string, dest string) (err error) {
 	}
 	return
 }
+
+// GetPvcNameWithSuffix returns a name given a base and a suffix
+func GetPvcNameWithSuffix(base, suffix string) string {
+	return getName(base, suffix, kvalidation.DNS1123SubdomainMaxLength, false)
+}
+
+// GetPodNameWithSuffix returns a name given a base and a suffix
+func GetPodNameWithSuffix(base, suffix string) string {
+	return getName(base, suffix, kvalidation.DNS1123SubdomainMaxLength, false)
+}
+
+// GetPodNameWithPrefix returns a name given a base and a suffix
+func GetPodNameWithPrefix(base, prefix string) string {
+	return getName(base, prefix, kvalidation.DNS1123SubdomainMaxLength, true)
+}
+
+func getName(base, affix string, maxLength int, asPrefix bool) string {
```

Comment:
why not import this in from openshift library-go?

If not importing in, may I suggest a new package called `naming`?

Reply:
Because I had to change it to keep our naming style, for import/upload pods we do add prefix:
"importer-${NAME}" and "cdi-upload-${NAME}", 
and for scratch we use 'scratch' as suffix:
"${NAME}-scratch".

Source pod is named differently - "UID-source-pod", so it was not changed in this PR.

New package sounds reasonable.

Reply:
We used uid-source-pod, with the uid being the uid of the target PVC to have a reproducible consistent name for the source pod, if we have a better way of naming it now, I am all for doing that.

Reply:
I have update to use openshift/library-go. 
I have left uid-source-pod not changed, since it is well within limits and is good acording to naming rules. The change to this and other names might be a "nice to do" thing - later, and will require special attention for backwards compatibility (like the usage of additional annotation).